packages/clarity-js/src/layout/schema.ts

```diff
@@ -24,7 +26,7 @@ export function ld(json: any): void {
                     break;
                 case JsonLD.AggregateRating:
                     if (json[JsonLD.RatingValue]) {
-                        metric.max(Metric.RatingValue, parseFloat(json[JsonLD.RatingValue]) * Setting.RatingScale);
+                        metric.max(Metric.RatingValue, num(json[JsonLD.RatingValue], Setting.RatingScale));
```

Comment:
IMPORTANT: Please do not do this in this PR for reasons I put in our convsation. Instead can you either leave it unscaled as now or create a seprate metric that has the scaled value and send both (for now) until we migrate over?